.github/workflows/test-coverage.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
```

Comment:
Should we remove the master branch from here?

Reply:
Didn't think it hurt but i have removed it now!